web/templates/popovers/user_group_info_popover.hbs

```diff
@@ -9,6 +9,15 @@
                 <div class="popover-group-menu-description">{{group_description}}</div>
             </div>
         </li>
+        {{#if members_count}}
+            <li role="none" class="popover-menu-list-item text-item italic">
+                {{#if (eq members_count 1)}}
+                <span class="popover-menu-label">{{t "1 member." }}</span>
+                {{else}}
+                <span class="popover-menu-label">{{t "{members_count} members." }}</span>
```

Comment:
I think this needs to be an ICU format plural so that it works in other languages that use different tense depending on more complex rules than "does it equal 1". Like in Russian 31 and 39 do different forms of "members".

Reply:
Fixed this.